src/Crate/PDO/PDO.php

```diff
@@ -42,6 +42,7 @@ class PDO extends BasePDO implements PDOInterface
 
     public const CRATE_ATTR_HTTP_BASIC_AUTH = 1000;
     public const CRATE_ATTR_DEFAULT_SCHEMA  = 1001;
+    public const CRATE_ATTR_BULK_MODE       = 1009;
```

Comment:
Firsthand, I thought it would be a good idea to introduce bulk mode operations, well, by adding another custom `CRATE_ATTR_BULK_MODE` item to the available custom PDO settings for CrateDB.

In general, it worked well so far to route this attribute through the layers of the machinery, in order to fulfil its main goal....